lib/manageiq/providers/workflows.rb

```diff
@@ -1,3 +1,6 @@
+require "manageiq/providers/workflows/engine"
+require "manageiq/providers/workflows/version"
```

Comment:
For engines, do we `require` our dependencies?

Reply:
yes, I don't think you want to autoload/reload engine/version